property_ext_test.go

```diff
@@ -0,0 +1,69 @@
+package restfulspec
+
+import (
+	"net"
+	"testing"
+)
+
+func TestThatExtraTagsAreReadIntoModel(t *testing.T) {
+	type fakeint int
+	type fakearray string
+	type Anything struct {
+		Name      string    `description:"name" modelDescription:"a test"`
+		Size      int       `minimum:"0" maximum:"10"`
+		Stati     string    `enum:"off|on" default:"on" modelDescription:"more description"`
+		ID        string    `unique:"true"`
+		FakeInt   fakeint   `type:"integer"`
+		FakeArray fakearray `type:"[]string"`
```

Comment:
To be honest, I assumed that you had added this `[]string` syntax in `go-restful-swagger12` yourself. I can take it out if you prefer.